modules/vaos/app/controllers/vaos/preferences_controller.rb

```diff
@@ -2,15 +2,29 @@
 
 module VAOS
   class PreferencesController < VAOS::BaseController
-    def index
-      response = preferences_service.get_preferences(current_user)
+    def show
+      response = preferences_service.get_preferences
+      render json: VAOS::PreferencesSerializer.new(response)
+    end
+
+    def update
+      response = preferences_service.put_preferences(put_params)
       render json: VAOS::PreferencesSerializer.new(response)
     end
 
     private
 
     def preferences_service
-      VAOS::PreferencesService.new
+      VAOS::PreferencesService.new(current_user)
+    end
+
+    def put_params
```

Comment:
It's more rails-y to call this either `preferences_params`. `put_params` to me sounds like I'm trying to put some params somewhere.

Reply:
I kind of prefer to have to have these as distinct lines, than chaining. Particularly because it's a little inconsistent. For example if I pass an array of keys to require, I can no longer chain permit.

Reply:
happy to change the name, but what if your method ends up being different for Update vs Create? In other words, if I need to distinguish then preferences_params would become what: `preferences_params_for_update` and `preferences_params_for_create`?